content/en/_index.html

```diff
@@ -11,6 +11,9 @@ <h1 class="text-center">Kubernetes Native Policy Management</h1>
 	</a>
 	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="/docs/introduction/#quick-start">
 		Get Started <i class="fa fa-arrow-alt-circle-right ml-2 "></i>
+	  </a>
+	  <a class="btn btn-lg btn-primary mr-3 mb-4" href="https://learn.nirmata.com/explore">
+		Get Certified <i class="fa fa-certificate ml-2 "></i>
```

Comment:
Hi Anita, I get the intent of showcasing this, but don't think this button is appropriate in this location. 

What I suggest is that we either add a "Support" page (like here https://www.openpolicyagent.org/support) and list the certification, and other offers there. What do you think?

Reply:
I think it is a great idea but I doubt support should be the name.

Reply:
If that's the case tho,  should the file be exiting under community?
or should it be a stand-alone file?

Reply:
It can also be a header under the resources tho

Reply:
We could create a "Training and Certification" section under the Community page and link to this plus Nirmata's [training course](https://learn.nirmata.com/courses/kyverno-in-production).

Reply:
Seems like we should create an area where other companies can also create PRs to list their commercial offers, distributions, trainings, services, etc. for Kyverno. 

If not "support" what would be a good short name for such a section?

Reply:
"Commercialization"?

Reply:
How about "Services"?

Reply:
May be worth checking with the CNCF on this. Will check if Kim has more information.

@Anita-ihuman - if you can split out the adopters program updates, we can merge those changes and use a separate PR for the certification changes.

Reply:
alright then will do that.